tests/test_icml_conference_with_templates.py

```diff
@@ -884,36 +885,27 @@ def test_post_submission(self, client, openreview_client, test_client, helpers,
         assert desk_reject_inv.expdate == expdate
         assert pc_client.get_invitation('ICML.cc/2025/Conference/-/PC_Revision')
 
-        ## make submissions visible to SACs, ACs only
+        # ## make submissions visible to SACs, ACs only
         
-        ac_client = openreview.api.OpenReviewClient(username = 'ac1@icml.cc', password=helpers.strong_password)
```

Comment:
Now we are not letting PCs change the submission_readers in `Submission_Change_Before_Bidding`, so SACs, ACS and reviewers are always added.

Reply:
Sounds good for now, we might want to give them the option to remove them if they want to enable bidding for ACs only, for example. But this is not needed for the first release.